ci/node.js.yml

```diff
@@ -17,10 +17,11 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install, build, and test
-      run: |
-        npm ci
-        npm run build --if-present
-        npm test
+    - name: install

```

Comment:
Think it's better here to just leave off the step names so that it uses run command for the step name

Reply:
Yep, leave out the `name`.

Reply:
✔️